src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -135,6 +144,14 @@ static ItemStack of(ItemType itemType, int quantity) {
      */
     boolean equalTo(ItemStack that);
 
+    /**
+     * Returns true if {@link #getQuantity()} is zero and therefore this
+     * ItemStack is empty.
```

Comment:
I meant more of: "What does an empty item stack differ from a 'non-empty' item stack? Does the client still render the item? Is it even an item to be considered by inventory logic? Can it be changed? etc."